sriov/common/utils.py

```diff
@@ -124,11 +125,104 @@ def rm_arp_entry(ssh_obj, ip):
     Raises:
         Exception: command failure
     """
-    cmd = f"arp -d {ip} || true"
+    cmd = f"arp -d {ip} || true"    # not a failure if the ip entry not exist
     code, _, err = ssh_obj.execute(cmd)
     if code != 0:
         raise Exception(err)
+
+def prepare_ping_test(tgen, tgen_intf, tgen_vlan, tgen_ip, tgen_mac, 
+                      dut, dut_ip, dut_mac,
+                      testdata):
+    """Collection of steps to prepare for ping test
+    
+    Args:
+        tgen: trafficgen ssh handler
+        tgen_intf(str): trafficgen physical interface name
+        tgen_vlan(int): vlan ID on the trafficgen physical interface
+        tgen_ip(str): trafficgen ip address
+        tgen_mac(str): trafficgen mac address; none will not add arp entry on DUT.
+        dut: DUT ssh handler
+        dut_ip: DUT ip address
```

Comment:
This an the mac below need types in the definition (I assume str)

Reply:
Ack